docs/wrapping-react/wrapping-react.md

```diff
@@ -0,0 +1,170 @@
+```python exec
+import reflex as rx
+from typing import Any
+from pcweb.base_state import State
+from pcweb.components.spline import spline
+from pcweb.templates.docpage import demo_box_style
+```
+# Wrapping React Overview
+
+
+One of Reflex's most powerful features is the ability to wrap React components. This allows us to build on top of the existing React ecosystem, and leverage the vast array of existing React components and libraries.
+
+If you want a specific component for your app but Reflex doesn't provide it, there's a good chance it's available as a React component. Search for it on [npm](https://www.npmjs.com/), and if it's there, you can use it in your Reflex app.
+
+In this section, we'll go over how to wrap React components on a high level. In the subsequent sections, we'll go over the details of how to wrap more complex components.
+
+
+## Spline Example
+
+
+Let's start with a library called [Spline](https://spline.design/). Spline is a tool for creating 3D scenes and animations. It's a great tool for creating interactive 3D visualizations.
+
+
+We have some code that creates a Spline scene. We want to wrap this code in a Reflex component so that we can use it in our Reflex app.
+
+```javascript
+import Spline from '@splinetool/react-spline';
+
+export default function App() {
+  return (
+    <Spline scene="https://prod.spline.design/up1SQcRLq1s6yks3/scene.splinecode" />
+  );
+}
+```
+
+The two most important props are `library`, which is the name of the npm package, and `tag`, which is the name of the React component.
+
+```python
+class Spline(rx.Component):
+    """Spline component."""
+
+    library = "@splinetool/react-spline"
+    tag = "Spline"
+    scene: Var[str] = "https://prod.spline.design/Br2ec3WwuRGxEuij/scene.splinecode"
+    is_default = True
```

Comment:
Maybe move the `is_default and `lib_dependencies` tags above `scene` so that the library props are together, and the "real" props are separated.